qt/python/mantidqt/widgets/plotselector/widget.py

```diff
@@ -0,0 +1,121 @@
+#  This file is part of the mantid workbench.
+#
+#  Copyright (C) 2018 mantidproject
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import absolute_import, print_function
+
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtGui import (QStandardItem, QStandardItemModel)
+from qtpy.QtWidgets import (QAbstractItemView, QListView, QWidget, QPushButton, QVBoxLayout, QHBoxLayout, QLineEdit)
+
+
+class PlotSelectorWidget(QWidget):
+    """
+    The view to the plot selector, a PyQt widget.
+    """
+    def __init__(self, presenter, parent=None):
+        """
+        Initialise a new instance of PlotSelectorWidget
+        :param presenter: The presenter that created this class
+        :param parent: A parent QWidget
+        """
+        QWidget.__init__(self, parent)
```

Comment:
We have used `super` elsewhere in the class, maybe use it here too for consistency?